app/views/foreman/unattended/finish-katello.erb

```diff
@@ -60,6 +60,8 @@ cat > /etc/puppet/puppet.conf << EOF
 <%= snippet 'puppet.conf' %>
 EOF
 
+<%= snippet('remote_execution_ssh_keys') %>
```

Comment:
It was moved to [community-templates](https://github.com/theforeman/community-templates), so it's bundled with Foreman since 1.11